sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -2032,6 +2032,10 @@ def create_pipeline_spec(
 
     platform_spec = pipeline_spec_pb2.PlatformSpec()
     if pipeline_config is not None:
+        workspace_config = pipeline_config.workspace
+        if workspace_config is not None:
```

Comment:
Sorry, I don't think the original issue was clear.

What we'd like to solve is when a user leverages something like `dsl.WORKSPACE_PATH_PLACEHOLDER` or another workspace feature in their component but they didn't set the workspace size or set an invalid value.

Reply:
Sure, I will update the changes.